corehq/apps/data_interfaces/utils.py

```diff
@@ -34,3 +35,32 @@ def add_cases_to_case_group(domain, case_group_id, uploaded_data):
         case_group.save()
 
     return response
+
+
+def archive_forms(user, uploaded_data):
+    response = {
+        'errors': [],
+        'success': [],
+    }
+
+    for row in uploaded_data:
+        xform_id = row.get('form_id')
+        try:
+            xform = XFormInstance.get(xform_id)
+        except Exception as e:
+            response['errors'].append(
+                _("Could not get XFormInstance {}: {}".format(xform_id, e)))
+            continue
+
+        xform_string = _("XFORM {} for domain {} by user '{}'").format(
+            xform['_id'],
+            xform['domain'],
+            user.username)
```

Comment:
Inconsistent use of `_("...").format(...)` (here) and `_("...".format(...))` (above). I'm not sure which is the right way, or if there is some other better way.

Also, your template strings are byte strings, but the values may (probably not in this case) be Unicode. It is safer/best practice to use Unicode strings.